src/main/scala/nl.knaw.dans.easy.solr4files/components/DDM.scala

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.solr4files.components
+
+import java.net.URL
+
+import nl.knaw.dans.easy.solr4files.SolrLiterals
+import nl.knaw.dans.easy.solr4files.components.DDM._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+
+import scala.xml.{ Node, XML }
+
+class DDM(xml: Node) extends DebugEnhancedLogging {
+
+  val accessRights: String = (xml \ "profile" \ "accessRights").text
+  val solrLiterals: SolrLiterals =
+    (xml \ "dcmiMetadata" \ "identifier").filter(isDOI).map(n => ("dataset_doi", n.text)) ++
+      (xml \ "profile" \ "creatorDetails").map(n => ("dataset_creator", n.text)) ++
+      (xml \ "profile" \ "creator").map(n => ("dataset_creator", n.text)) ++
+      (xml \ "profile" \ "title").map(n => ("dataset_title", n.text)) ++
+      (xml \ "profile" \ "audience").flatMap(n => Seq(
+        ("dataset_narcis_audience", n.text), // TODO https://github.com/DANS-KNAW/easy-dtap/pull/131#pullrequestreview-60582037
+        ("dataset_audience", audienceMap.getOrElse(n.text, ""))
+      )) ++
+      (xml \ "dcmiMetadata" \ "subject").flatMap { n =>
+        val abrMap = getAbrMap(n) // FIXME
+        if (abrMap.isEmpty) Seq(("dataset_subject", n.text))
+        else {
+          Seq(
+            ("dataset_abr_subject", n.text), // TODO https://github.com/DANS-KNAW/easy-dtap/pull/131#pullrequestreview-60582037
+            ("dataset_subject", abrMap.get.getOrElse(n.text, ""))
+          )
+        }
+      } ++
+      (xml \ "dcmiMetadata" \ "relation").map(n => ("dataset_relation", n.text))
+  (xml \ "dcmiMetadata" \ "conformsTo").map(n => ("dataset_relation", n.text))
```

Comment:
@PaulBoon are you sure you want all these subtypes of relations be marked as `dataset_relation`? You don't need these subtypes?